src/wx/opts.cpp

```diff
@@ -564,7 +566,7 @@ void load_opts()
             cfg->Read(opt.opt, &opt.curint, *opt.intopt);
 
             if (opt.curint < opt.min || opt.curint > opt.max) {
-                wxLogWarning(_("Invalid value %d for option %s; valid values are %d - %d"), opt.curint, opt.opt.c_str(), opt.min, opt.max);
+                wxLogWarning(_("Invalid value %d for option %s; valid values are %d - %d"), opt.curint, opt.opt.c_str(), int(opt.min), int(opt.max));
```

Comment:
Using modern C++ casts, +1

Reply:
I know this is stupid, but this was giving a warning->segfault message on wxWidgets `3.0`->`3.1` on Windows 10. That is the reason here =P